data/abilities.js

```diff
@@ -744,7 +744,7 @@ let BattleAbilities = {
 	},
 	"disguise": {
 		desc: "If this Pokemon is a Mimikyu, the first hit it takes in battle deals 0 neutral damage. Its disguise is then broken, it changes to Busted Form, and it loses 1/10 of its max HP. Confusion damage also breaks the disguise.",
-		shortDesc: "If this Pokemon is a Mimikyu, the first hit it takes in battle deals 0 neutral damage.",
+		shortDesc: "If this Pokemon is a Mimikyu, first hit 0 damage. Breaks disguise losing 10% HP",
```

Comment:
```suggestion
		shortDesc: "(Mimikyu only) The first hit it takes is blocked, and it takes 10% HP damage instead.",
```

Reply:
Sure, do we need it for every ability that changed in Gen 8 or just Disguise?

Reply:
We've already done it for other Abilities:

https://dex.pokemonshowdown.com/abilities/moody

Reply:
Feel free to do it for any ability we've missed, though?